app/assets/stylesheets/application.scss

```diff
@@ -11,8 +11,20 @@
  * file per style scope.
  *
  *= require_self
- *= require_tree .
  */
 
-@import "bootstrap-sprockets";
-@import "bootstrap";
+/* Font Awesome Imports */
+ @import "font-awesome-sprockets";
+ @import "font-awesome";
+
+ /* Site Sass Imports */
```

Comment:
I've been focusing on Frontend Design lately and this has been a really easy way to keep things under control, especially for bug fixes. Seems like a lot of designers use similar structures for Sass development. Thought it would be a good way to show students a simple way of keeping styling clean and componentized.